tests/integration/e2e.spec.js

```diff
@@ -6,7 +6,9 @@ let client;
 
 beforeAll(() => {
   // Set your credentials!
+  // eslint-disable-next-line jest/no-standalone-expect
```

Comment:
According to https://github.com/jest-community/eslint-plugin-jest/blob/main/docs/rules/no-standalone-expect.md, `expect` statements do not get executed outside of tests, but it doesn't mention `beforeAll`. Would this rule still trigger if you use a helper action?

Reply:
Yep. This rule is triggered when using `expect` in helper action.